resources/sass/app.scss

```diff
@@ -242,6 +250,14 @@ legend {
     }
 }
 
+.list-bios {
+    @extend %index-list;
```

Comment:
well, it's not that one is extending another; rather, there's an index-list base class that the other lists basically extend. It's just a lazy way to avoid having to do `class="index-list bios-list"` and then later `index-list conferences-list`, etc.

`index-list` is the class that's independent of the content, and then it's just mildly extended in each context.

Thoughts?